validmind/unit_metrics/__init__.py

```diff
@@ -0,0 +1,192 @@
+import pandas as pd
+import json
+import hashlib
+import importlib
+
+from ..utils import get_model_info
+
+
+global_metric_values = {}
```

Comment:
```suggestion
unit_metric_results_cache = {}
```

^ Just to make it explicit that we use it as a simple cache of unit metric results.